mitodl/mitxpro
# feat: add ondemand webinar body text field

#### Pre-Flight checklist


#### What are the relevant tickets?
Followup for https://github.com/mitodl/hq/issues/1818#issuecomment-1682256526

#### What's this PR do?
Adds a new field for in webinar model. Also, it makes some enhancements to the webinar card links.

#### How should this be manually tested?

- Create a couple of upcoming and on-demand webinars
- Verify that a new field named `body_text` is added for webinars and visible in CMS.
- body_text is also visible under the webinar recording video
- go to the webinar list page i.e. `/webinars/`
- Verify that the on-demand webinar title link and view recording button redirect to the webinar detail page.
- Verify that the title and register button links redirect to the action_url link added through cms for the upcoming webinars. Links should not redirect to the detail page. The detail page for an upcoming webinar would still be accessible by manually entering a URL.

#### Screenshots (if appropriate)
<img width="1734" alt="Screen Shot 2023-08-21 at 2 39 56 PM" src="https://github.com/mitodl/mitxpro/assets/52656433/0719a4d8-d592-4739-ab09-eab2e31c1b52">
